Documentation/CRDs/Shared-Filesystem/ceph-filesystem-crd.md

```diff
@@ -133,15 +133,18 @@ spec:
     enabled: true
     # list of Kubernetes Secrets containing the peer token
     # for more details see: https://docs.ceph.com/en/latest/dev/cephfs-mirroring/#bootstrap-peers
```

Comment:
Actually there is a similar example in [filesystem.yaml](https://github.com/rook/rook/blob/master/deploy/examples/filesystem.yaml#L113-L122) if you could update that as well